chalice/local/internal.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+"""
+This is intended only for `pytest-chalice`.
+
+https://github.com/studio3104/pytest-chalice
+"""
+
+import json
+import re
+
+from logging import getLogger
+from typing import Any, Callable, Dict, Optional  # noqa
+
+from chalice import Chalice  # noqa
+from chalice.config import Config
+from chalice.local import LocalGateway
+from chalice.local import EventType, HeaderType  # noqa
+
+logger = getLogger(__name__)
+
+# Python 3.4 or older don't have JSONDecodeError within json module
+try:
+    JSONDecodeError = json.JSONDecodeError  # type: ignore
+except AttributeError:
+    JSONDecodeError = ValueError  # type: ignore
```

Comment:
I'm ok with just always catching `ValueError` instead of `JSONDecodeError` to keep things simple.

Reply:
Updated https://github.com/aws/chalice/pull/1193/commits/1adab1be1390c4c4ac6f2117ad3e27f836457a62